packages/react-aria-components/stories/Table.stories.tsx

```diff
@@ -392,3 +393,134 @@ const MyCheckbox = ({children, ...props}: CheckboxProps) => {
     </Checkbox>
   );
 };
+
+const MyTableLoader = () => {
+  return (
+    <UNSTABLE_TableLoader>
+      {({isTableEmpty}) => (
+        <span>
+          {isTableEmpty ? 'Loading spinner' : 'Load more spinner'}
```

Comment:
If we recommend using `renderEmptyState` to show the initial loading spinner, do we need this `isTableEmpty` render prop?

Reply:
we wouldn't need `isTableEmpty` any more, this loader would then be specifically for when we are in a "load more" case/for loading nested row loaders. However, it feels a bit confusing that the user would use `renderEmptyState` to render the initial loading spinner IMO since then they wouldn't use `UNSTABLE_TableLoader` in their `renderEmptyState` call, they would just pass it a div or something since https://github.com/adobe/react-spectrum/blob/10bac541a2a5c666b443fe70f313455a405a5c77/packages/react-aria-components/src/Table.tsx#L856-L864 wraps it in a row already. Also feels confusing that the end use would still need to render the  `UNSTABLE_TableLoader` for the `loadMore` case still as well